src/lib/middleware/local_assets.coffee

```diff
@@ -0,0 +1,32 @@
+#
+# Serves client-side assets from Force, or from CDN
+#
+
+{ CDN_URL } = require '../../config'
+fs = require 'fs'
+path = require 'path'
+
+manifestPath = path.join(__dirname, '..', '..', '..', 'public', 'assets', 'manifest.json')
+AssetMap = if fs.existsSync(manifestPath) then require(manifestPath) else {}
+
+module.exports = (req, res, next) ->
+  res.locals.asset = (filename) ->
```

Comment:
This replaces the `asset` helper to return a corresponding fingerprinted file in the `manifest.json` if it exists, otherwise it just returns the original filename. We can add the `CDN_URL` support here too.